packages/core/admin/admin/src/components/Notifications/Notification/index.js

```diff
@@ -67,7 +67,14 @@ const Notification = ({ dispatch, notification }) => {
   }
 
   if (title) {
-    alertTitle = typeof title === 'string' ? title : formatMessage(title);
+    alertTitle =
+      typeof title === 'string'
+        ? title
+        : formattedMessage({
+            id: title?.id || title,
+            defaultMessage: title?.defaultMessage || title?.id || title,
```

Comment:
1. Why do you need to id as default message fallback?
2. Wouldn't the title being a string case already handled by `title === 'string'` ?